src/commands/copy/copy-worker.ts

```diff
@@ -9,6 +10,29 @@ import { CopyContract, CopyContractArgs, CopyStats } from './copy-rpc.js';
 
 const Q = new ConcurrentQueue(10);
 
+export const FixableContentType = new Set(['binary/octet-stream', 'application/octet-stream']);
+
+/** If the file has been written with a unknown binary contentType attempt to fix it with common content types */
+export function fixFileMetadata(target: string, f: FileInfo): FileInfo {
+  // If the content is encoded we do not know what the content-type should be
+  if (f.contentEncoding != null) return f;
+  if (!FixableContentType.has(f.contentType ?? 'binary/octet-stream')) return f;
+
+  if (target.endsWith('.tiff') || target.endsWith('.tif')) {
```

Comment:
Could use the existing `isTiff()` that also test `.TIFF` or `.TIF`. Even if it would most likely not happen if the file has been standardised before.